perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -198,7 +198,7 @@ def FillDevice(vm, disk, fill_size):
               '--rw=write --direct=1 --size=%s') %
              (fio.FIO_PATH, disk.GetDevicePath(), fill_size))
 
-  vm.RemoteCommand(command)
+  vm.RobustRemoteCommand(command)
```

Comment:
I didn't turn on logging here because this is a pretty boring command that just pre-fills the disk. I do use logging in the more interesting call to RobustRemoteCommand later.